backend/api_v2/management/commands/delete_org_rate_limit.py

```
@@ -0,0 +1,103 @@
+class Command(BaseCommand):
+    help = "Delete custom organization rate limit (reverts to default)"
+
+    def add_arguments(self, parser):
+        parser.add_argument("org_id", type=str, help="Organization ID or name")
+        parser.add_argument(
+            "--force",
+            action="store_true",
+            help="Skip confirmation prompt",
+        )
+
+    def handle(self, *args, **options):
```

Ruff (0.14.3)

Unused method argument: `args`

(ARG002)